### PR TITLE
Initial documentation about how to write a proposal

### PR DESCRIPTION
**Pull request by AliSoftware**
This document explains how our process to write a proposal works

With ❤️

### CODE REVIEW

File: Cookbook/Proposals/Template_Proposal_Meeting_Invitation.md [hunk 1]
```diff
@@ -1,27 +1,32 @@
 Hi all,
 
-This meeting is to discuss the "<Proposal Title>" proposal authored by <Proposal's Author>.
+This meeting is to discuss the "<#Proposal Title#>" proposal authored by <#Proposal's Author#>.
```
Comment (AliSoftware) [inline Cookbook/Proposals/Template_Proposal_Meeting_Invitation.md]: Had to add those because otherwise GitHub sees `<xx yy>` as if it were an HTML tag and doesn't render the text inside, so that part was not rendered and thus not copied during copy/paste in Outlook

File: Cookbook/Technical-Documents/WritingAProposal.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+### Write a proposal document
+
+* Create a copy of [the proposal template](../Proposals/Template_Proposal.md)
+* Fill all the fields and paragraphs, describing your proposal the best you can.
+* Be sure to be clear in your explanation of the problem to solve, the proposed solutions, but also potential limitations and the alternatives considered
+
+### Make a Pull Request
+
+* Describe in a few words the purpose of the proposal in the PR body. Also mention any previous discussion (e.g. Slack) on the topic.
+* Copy-paste [the template dedicated to proposal PRs](https://raw.githubusercontent.com/Babylonpartners/ios-playbook/master/.github/PULL_REQUEST_TEMPLATE/proposal.md) at the end and fill it
```
Comment (ilyapuchka) [inline Cookbook/Technical-Documents/WritingAProposal.md]: or add a `template=proposal.md` query parameter to the URL when you are creating a pull request so that it is pasted for you in the pull request body automatically

File: Cookbook/Technical-Documents/WritingAProposal.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+After the proposal PR has been posted, your coworkers have two weeks to read it and make comments on it and get the discussion started before we have a zoom meeting to discuss it and decide if it's accepted or rejected.
+
+* Create a new Outlook invite using the [meeting invitation template](../Proposals/Template_Proposal_Meeting_Invitation.md) to fill the body, replacing the placeholder fields accordingly
+* Schedule that meeting for 2 weeks from now
+* Send the invite to our `ios-developers` mailing list
+
+
+### Have the meeting
+
+* Don't forget to create the zoom meeting and update the Outlook invite with the woom link before the meeting
```
Comment (ilyapuchka) [inline Cookbook/Technical-Documents/WritingAProposal.md]:
```suggestion
* Don't forget to create the zoom meeting and update the Outlook invite with the Zoom link before the meeting
```

File: Cookbook/Technical-Documents/WritingAProposal.md (hunk 2 above)
Comment (AliSoftware) [reply Cookbook/Technical-Documents/WritingAProposal.md]: @ilyapuchka That was my original idea too, but to create the URL you need to also know the name of the branch 😢 

`https://github.com/Babylonparners/ios-playbook/compare/master...NAMEOFBRANCH?quick_pull=1&template=proposal.md`

**Review by inamiy — APPROVED** [review]
👏

File: Cookbook/Proposals/Template_Proposal_Meeting_Invitation.md [hunk 4]
```diff
@@ -1,27 +1,32 @@
...
-2.	Actual proposal here <Link to the Proposal's .md file>.
+1.	PR here <#Link to the Proposal's PR#>.
+2.	Actual proposal here <#Link to the Proposal's .md file#>.
 
 Meeting Etiquette:
 
 1.	**Please fully read the proposal.**
-  a.	If you haven’t read the proposal, or don’t care for the outcome, please don’t show up for the meeting.
+
+   * If you haven’t read the proposal, or don’t care for the outcome, please don’t show up for the meeting.
```
Comment (diegopetrucci) [inline Cookbook/Proposals/Template_Proposal_Meeting_Invitation.md]: I feel like this is a bit too aggressive. Would rephrase to something like

```suggestion
   * If you are not interested in participating in the proposal discussion you are free to skip the meeting. If you are, please make sure to have read the document and its discussion carefully before the meeting starts, as the proposal itself will not be explained again.
```

**Review by diegopetrucci — APPROVED** [review]
This is very nice

File: Cookbook/Proposals/Template_Proposal_Meeting_Invitation.md (hunk 4 above)
Comment (AliSoftware) [reply Cookbook/Proposals/Template_Proposal_Meeting_Invitation.md]: @diegopetrucci I agree about changing the wording, but I read your suggestion as "[...] you are free to skip the meeting. If you are **[skipping the meeting]**, please make sure to have read [...]" which was confusing 😅 open for suggestions on how to improve that

Comment (AliSoftware) [reply Cookbook/Proposals/Template_Proposal_Meeting_Invitation.md]: @diegopetrucci how about:
```suggestion
   * Please make sure to have read the document and its discussion carefully before the meeting starts, as the proposal itself will not be explained again.
   * If you are not interested in participating in the proposal discussion, you are free to skip the meeting.
```

Comment (diegopetrucci) [reply Cookbook/Proposals/Template_Proposal_Meeting_Invitation.md]: go ahead with yours 🚀
